testtracer_test.go

```diff
@@ -39,10 +46,39 @@ func (n testTracer) StartSpanWithOptions(opts StartSpanOptions) Span {
 
 // Inject belongs to the Tracer interface.
 func (n testTracer) Inject(sp Span, format interface{}, carrier interface{}) error {
-	return nil
+	span := sp.(testSpan)
+	switch format {
+	case TextMap:
+		// Just for testing purposes... generally not a worthwhile thing to
+		// propagate.
+		carrier.(TextMapWriter).Add(testHTTPHeaderPrefix+"opname", span.OperationName)
+		carrier.(TextMapWriter).Add(testHTTPHeaderPrefix+"hasparent", strconv.FormatBool(span.HasParent))
```

Comment:
I guess so... it's "just a test" and all of that. I can add some other junk to the spans for this purpose, though.